thunx-pdp-opa/src/main/java/com/contentgrid/thunx/pdp/opa/QuerySetToThunkExpressionConverter.java

```diff
@@ -57,6 +63,7 @@ public ThunkExpression<Boolean> convert(Query query) {
 
         var expressions = query.stream()
                 .map(this::convert)
+                .filter(Objects::nonNull)
```

Comment:
This filtering is done because there is an ability to return empty Term for in operation. 
Example:
`
                    "index": 4,
                    "terms": [
                        {
                            "type": "ref",
                            "value": [
                                {
                                    "type": "var",
                                    "value": "internal"
                                },
                                {
                                    "type": "string",
                                    "value": "member_2"
                                }
                            ]
                        },
                        {
                            "type": "ref",
                            "value": [
                                {
                                    "type": "var",
                                    "value": "input"
                                },
                                {
                                    "type": "string",
                                    "value": "entity"
                                },
                                {
                                    "type": "string",
                                    "value": "accountId"
                                }
                            ]
                        },
                        {
                            "type": "array",
                            "value": [] # will return null in conversion, cause we need to skip empty in clause
                        }
                    ]
                }`

Reply:
That's why in this place we are filtering empty `in` cases